trp_project_scrum/security/res_groups.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="external_scrum_users" model="res.groups">
+        <field name="name">External Users - Scrum</field>
+        <field name="is_portal">True</field>
+        <field
+            name="implied_ids"
+            eval="[(6, 0, [
+            ref('trp_external_user.group_external_user'),
+            ref('base.group_portal')])]"/>
```

Comment:
i checked if portal group gave you something you where missing above. i see not much in base.
You cn do a quick analysis of what your implied groups give you in terms of access via interface , or by 
`grep base.group_portal | grep csv `

obvioulsy conly considering the dependency tree of your current module.
for example i see:

website_hr/security/ir.model.access.csv:access_hr_employee_portal,hr.employee.portal,hr.model_hr_employee,base.group_portal,1,0,0,0

but we don't depend on hr_website so you got nothing
I see nothing , so you may have to add your missing access, but check nevertheless.

adding another access cannot do no harm. If it is more restricive than another existing one, it will be ignored.